src/Controllers/responseController.ts

```diff
@@ -4,4 +4,8 @@ import { Response } from 'express';
 
 export const response = (res: Response<unknown, Record<string, unknown>>, response: Result<ApiResponse, BaseError>, statusCodeSuccess: number) => {
     res.status(response.success ? statusCodeSuccess : Number(response.error.statusCode)).send(response.success ? { success: response.success, data: response.result.data } : { success: response.success, message: response.error.message });
+};
+
+export const responseError = (res: Response<unknown, Record<string, unknown>>, response: Result<ApiResponse, BaseError>) => {
+    res.status(response.success ? 400 : Number(response.error.statusCode)).send(response.success ? { success: response.success, data: response.result.data } : { success: response.success, message: response.error.message });
```

Comment:
Hvad er det som bliver tjekket på her? Hvis vi bare skal smide en statuscode for at smide en, foreslår jeg 500.